### DISCUSSION TITLE
Call for maintainers 🗣️📣

### DISCUSSION THREAD

**Discussion by abevoelker** [abevoelker/devise-passwordless#64] [Announcements]
Hi all,

I wrote this library to solve a specific problem in a Rails app I was developing. The library currently meets that need, but there are still bugs to fix and more features worth developing.

Unfortunately for the foreseeable future I won't have much time to invest in working on this library nor closely overseeing feature development. I don't want to abandon the project but due to some life issues I have to put this at the bottom of my priority heap. I'd love someone (or some people) to step up and help maintain it going forward.

So if you're reading this and interested, reply here or you can private message me via email (abe@abevoelker.com) or other social links. All I ask is that you've contributed a PR in the past, or opened an issue, or can provide a similar proof of competency.

Thanks! :v:

**Comment by moritonal** [comment]
Hey Abe, my name's [Tom](https://www.linkedin.com/in/moritonal/) and we use this package over at my company. Just wanted to say thanks for building this project, hope your time clears up for the better, and wanted to offer whatever help I can in support or maintenance.
